STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/inc/_az_ulib_dm_package.h

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+#ifndef _AZ_ULIB_DM_PACKAGE_H
+#define _AZ_ULIB_DM_PACKAGE_H
+
+#include "az_ulib_ipc_interface.h"
+#include "az_ulib_result.h"
+#include "azure/az_core.h"
+
+#ifndef __cplusplus
+#include <stdint.h>
+#else
+#include <cstdint>
+#endif /* __cplusplus */
+
+#include "azure/core/_az_cfg_prefix.h"
+
+/**
+ * @brief Package shell entry point expected signature.
+ *
+ * @param[in]   code    The pointer to the code.
+ */
+typedef void (*_az_ulib_dm_package_shell_entry_point)(void* code);
+
+/**
+ * @brief Publish interface expected signature.
+ *
+ * @param[in]   table   The pointer to #az_ulib_ipc_table with the IPC functions.
+ *
+ * @return The #az_result with the publish result.
+ *  @retval #AZ_OK                  If the interface was published with success.
+ *  @retval non-AZ_OK               If there was any issue publishing the interface.
+ */
+typedef az_result (*_az_ulib_dm_package_publish_interface)(const az_ulib_ipc_table* const table);
+
+/**
+ * @brief Unpublish interface expected signature.
+ *
+ * @param[in]   table   The pointer to #az_ulib_ipc_table with the IPC functions.
+ *
+ * @return The #az_result with the publish result.
+ *  @retval #AZ_OK                  If the interface was unpublished with success.
+ *  @retval non-AZ_OK               If there was any issue unpublishing the interface.
+ */
+typedef az_result (*_az_ulib_dm_package_unpublish_interface)(const az_ulib_ipc_table* const table);
+
+/** Define the Module ID, which is used to indicate a module is valid.  */
+#define _AZ_ULIB_DM_PACKAGE_ID 0x4D4F4455
```

Comment:
Where did this value come from?

Reply:
It is a magic number defined by ThreadX Module.